atomic_reactor/utils/operator.py

```diff
@@ -276,11 +430,17 @@ def _container_pullspecs(self):
         ]
 
     def _annotation_pullspecs(self):
+        # Known sources of pullspecs in annotations
+
+        # TODO: Should this be a thing? Have known sources of pullspecs in annotations
+        # and handle them separately from the unknown ones? If so, should only the top
+        # level .metadata.annotations be considered here, or all the nested ones also?
```

Comment:
Not sure if it is worth handling the "containerImage" annotation separately from the other (heuristic) annotations, WDYT?

Reply:
IMO yes, this is not heuristic, it's clear that this annotation must contain pullspec.

Reply:
no

Reply:
Ok, I was thinking the same thing. What about the second point? Is "containerImage" only expected in the top-level metadata.annotations, or potentially also other metadata.annotations in the file?

Reply:
I'll collect potential "containerImage" keys from all metadata.annotations objects and ignore "containerImage" in the heuristic, that way we should not find the same annotation pullspec twice